apps/etterlatte-behandling/src/main/kotlin/oppgaveny/OppgaveRoutesNy.kt

```diff
@@ -10,14 +10,18 @@ import io.ktor.server.routing.post
 import io.ktor.server.routing.put
 import io.ktor.server.routing.route
 import no.nav.etterlatte.Kontekst
+import no.nav.etterlatte.libs.common.SAKID_CALL_PARAMETER
 import no.nav.etterlatte.libs.common.kunSaksbehandler
 import no.nav.etterlatte.libs.common.oppgaveNy.FjernSaksbehandlerRequest
 import no.nav.etterlatte.libs.common.oppgaveNy.OppgaveNy
 import no.nav.etterlatte.libs.common.oppgaveNy.OpprettNyOppgaveRequest
 import no.nav.etterlatte.libs.common.oppgaveNy.RedigerFristRequest
 import no.nav.etterlatte.libs.common.oppgaveNy.SaksbehandlerEndringDto
 
-internal fun Route.oppgaveRoutesNy(service: OppgaveServiceNy, kanBrukeNyOppgaveliste: Boolean) {
+internal fun Route.oppgaveRoutesNy(
+    service: OppgaveServiceNy,
+    kanBrukeNyOppgaveliste: Boolean
+) {
     route("/api/nyeoppgaver") {
```

Comment:
Kan gi meining å dele i to routes på toppnivå her - ei for oppgåvelista og ei for det som skjer i kontekst av ei eksisterande sak. Da kan også parameteren ligge definert på toppen for sistnemnte.

Reply:
Ja slik var det før men vi ville at det skulle være en og samme handling for vedtak slik at man kan kjøre en intransaction på begge disse to operasjonene.

Reply:
Godt poeng!